src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { OrchestratorService } from '~/app/shared/api/orchestrator.service';
+import { SettingsService } from '~/app/shared/api/settings.service';
+import { NotificationType } from '~/app/shared/enum/notification-type.enum';
+import { NotificationService } from '~/app/shared/services/notification.service';
+
+@Component({
+  selector: 'cd-create-cluster',
+  templateUrl: './create-cluster.component.html',
+  styleUrls: ['./create-cluster.component.scss']
+})
+export class CreateClusterComponent implements OnInit {
+  constructor(
+    private orchService: OrchestratorService,
+    private router: Router,
+    private settingsService: SettingsService,
+    private notificationService: NotificationService
+  ) {}
+
+  ngOnInit() {
+    this.orchService.status().subscribe((status) => {
+      if (!status.available && !status.features.add_host.available) {
+        this.router.navigate(['']);
+      }
+    });
+  }
```

Comment:
You already have a `ModuleStatusGuardService` activation guard that blocks routing based on orchestrator availablity. I'd rather reuse that.

Reply:
I tried to reuse it but with the ModuleStatusGuardService it redirects it to the error page right? I've tried to redirect it to /dashboard page but couldn't. So without the current change e2e was failing for login and logout.